docs/oidc_testing/examples/.gitignore

```diff
@@ -0,0 +1,2 @@
+config-dev.yaml
+dex.db
```

Comment:
This should be dropped. It was mentioned elsewhere that this file is no longer present.

Reply:
Good catch.  That file gets created dynamically when the test container is started up.  It's in .gitignore now so that someone who does some testing won't accidentally add their test database, potentially with a sensitive password, and so it doesn't show up in "git status" as untracked files. :)

Reply:
That reasoning makes sense. Thanks.